FESS-GUI/graph.cpp

```diff
@@ -84,15 +85,21 @@ void ScrollingTimeGraph::addData(QCustomPlot* plot, double time, double primaryD
     plot->graph(3)->addData(time, secondaryData);
 
     // remove data of lines that's outside visible range:
-    plot->graph(0)->removeDataBefore(time-8);
-    plot->graph(1)->removeDataBefore(time-8);
+    plot->graph(0)->removeDataBefore(time-range);
+    plot->graph(1)->removeDataBefore(time-range);
 
     // rescale value (vertical) axis to fit the current data:
-    plot->graph(0)->rescaleValueAxis();
-    plot->graph(1)->rescaleValueAxis(true);
+    if(maxValue < 0){
```

Comment:
Currently, it is possible that maxValue will have a negative value. We should look into fixing that. There are some weird consequences. I'll bring it up in the meeting